### PR TITLE
Adds 2 new cave POIs

### PR DESCRIPTION
**Pull request by BillyBangles**
There's a few neat things here.
- A POI in the caves: a mysterious abandoned temple
- A candelabra!
- A variant of candelabras and normal candles that lights itself when it's spawned and never goes out (unless put out by a player)
- A neat red crystal that looks like supermatter but isn't (might need a sprite change if this is a problem)
- A dark ritual.

Screenshots:
https://puu.sh/zhZjV/8146dc460c.png
https://puu.sh/zi9rU/319da8303f.png

(Tested, all works.)

### CODE REVIEW

**Comment by Atermonera** [comment]
That diff. Praise be to Map Merger! Praise be to Satan!

**Comment by Anewbe** [comment]
Uh... Yeah, why was the main map touched?

**Comment by Atermonera** [comment]
Looking just at the numbers on the diff, I'd guess that you removed something somewhere and it shifted the index for everything by -1 Unfortunately github is too scared to show the full diff, as bad an idea as that is

**Comment by BillyBangles** [comment]
i do not recall touching the main map. i have no idea what i did to fuck that up.

**Comment by BillyBangles** [comment]
alright that looks like it did it

File: code/game/objects/items/weapons/candle.dm [hunk 1]
```diff
@@ -70,3 +70,42 @@
...
+		src.lit = 1
+		//src.damtype = "fire"
+		for(var/mob/O in viewers(usr, null))
+			O.show_message("<span class='notice'>\The [name] mysteriously lights itself!.</span>", 1)
+		set_light(CANDLE_LUM)
+		processing_objects.Add(src)
+
+/obj/item/weapon/flame/candle/candelabra/everburn
+	wax = 99999
+	light()
```
Comment (Anewbe) [inline code/game/objects/items/weapons/candle.dm]: Uh... This doesn't actually do anything, as far as I can tell? And I'm also mildly surprised it actually compiles with this here. Could be wrong.

Comment (BillyBangles) [reply code/game/objects/items/weapons/candle.dm]: ah, right, forgot to take that bit out, since i made it light itself in new() instead

**Comment by BillyBangles** [comment]
that should do it

**Pull request merged**